client/src/client.rs

```diff
@@ -1848,6 +1843,33 @@ impl<B, E, Block, RA> BlockBody<Block> for Client<B, E, Block, RA>
 			_ => None,
 		})
 	}
+
+	fn block_status(&self, id: &BlockId<Block>) -> sp_blockchain::Result<BlockStatus> {
+		// this can probably be implemented more efficiently
```

Comment:
Was this comment copied from somewhere? Can you elaborate what we could do to improve it here? Or maybe create a separate issue?

Reply:
Yeah the comment was copied from the original function body